dts/bindings/flash_controller/nordic,nrf-qspi.yaml

```diff
@@ -39,3 +39,8 @@ properties:
 
   pinctrl-0:
     required: true
+
+  timeout:
+    type: int
+    default: 500
+    description: Timeout for QSPI operations in ms.
```

Comment:
First, this seems to be a software setting, ie, does not belong to DT.

But it also has DT issues:
- Not ussing suffix (-ms)
- Default not justified (random value here)